cli.js

```diff
@@ -17,7 +17,8 @@ var argv = require('minimist')(process.argv.slice(2), {
     delay: ['d'],
     poll: ['p'],
     outpipe: ['o'],
-    filter: ['f']
+    filter: ['f'],
+    'await-write-finish': ['F']
```

Comment:
Is there a better flag to use (i.e. `-a`), or maybe only support the long flag? It shouldn't be a highly recommended feature to use (it's discouraged in chokidar).

Reply:
Yes, I think the only support for the long flag will be ok, as it's a highly optional feature for rare cases. Will update it.

Reply:
Updated 👌